lib/AST/DeclContext.cpp

```diff
@@ -271,6 +272,37 @@ ModuleDecl *DeclContext::getParentModule() const {
   return const_cast<ModuleDecl *>(cast<ModuleDecl>(DC));
 }
 
+/// Find the import that makes the given nominal declaration available.
```

Comment:
Could you move this doc comment onto the declaration in the .h file? It should also be tweaked to say "module" instead of "nominal declaration".